backend/src/nodes/properties/inputs/file_inputs.py

```diff
@@ -105,6 +107,28 @@ def PthFileInput(primary_input: bool = False) -> FileInput:
     )
 
 
+def CkptFileInput(primary_input: bool = False) -> FileInput:
+    """Input for submitting a local stable diffusion checkpoint file"""
+    return FileInput(
+        input_type_name="CkptFile",
+        label="Stable Diffusion Checkpoint",
+        file_kind="ckpt",
+        filetypes=[".ckpt", ".safetensors", ".pth"],
+        primary_input=primary_input,
+    )
+
+
+def StableDiffusionPtFileInput(primary_input: bool = False) -> FileInput:
+    """Input for submitting a local stable diffusion CLIP or VAE model file"""
+    return FileInput(
+        input_type_name="StableDiffusionPtFile",
+        label="Model",
+        file_kind="pt",
+        filetypes=[".ckpt", ".pt", ".bin", ".pth", ".safetensors"],
+        primary_input=primary_input,
+    )
```

Comment:
Is there any reason these can't just be all combined into the existing Load Model node? At least eventually? IMO it would be ideal to just have one node that can load all these file types and figure out based on model metadata what kind of thing it is. For example right now we don't have different Load Model nodes for each arch we support, we have one that loads any .pth or .pt and figures out which arch it is based on the contents of the model. (This puts the burden of figuring out what the model is on chaiNNer rather than on the user). Plus, any type restrictions off of them can be done still via Navi types.

Reply:
I think it's different from having different arches, since each of the model types (SD, CLIP, VAE) is used in a different set of nodes.  There's no "apply model" node that works with any of them, like there is for the pytorch models.  We could still one one type that covers everything and do the rest with type properties or whatever.  I have no opinion either way.

We could have one "Load Model" node that loads any model and returns the appropriate type.  That will come in handy when there's more types of models to load:  LORAs, controlnet things, etc.

The only complication, and the reason there's two different inputs here, is that "Checkpoints" actually have three different models in, so I think need a separate node.  Checkpoints seem to have one set of possible extensions, and others (VAEs, CLIPs, LORAs, etc) have another.